crates/compiler/test_gen/benches/list_map.rs

```diff
@@ -10,11 +10,16 @@ use roc_gen_llvm::{llvm::build::LlvmBackendMode, run_roc::RocCallResult, run_roc
 use roc_mono::ir::OptLevel;
 use roc_std::RocList;
 
-// results July 6th, 2022
+// results July 6, 2022
 //
 //    roc sum map             time:   [612.73 ns 614.24 ns 615.98 ns]
 //    roc sum map_with_index  time:   [5.3177 us 5.3218 us 5.3255 us]
 //    rust (debug)            time:   [24.081 us 24.163 us 24.268 us]
+//
+// results April 9, 2023
+//
+//    roc sum map             time:   [510.77 ns 517.47 ns 524.47 ns]
+//    roc sum map_with_index  time:   [573.49 ns 578.17 ns 583.76 ns]
```

Comment:
the native roc `map_with_index` got a lot faster, so that's nice! A pure roc implementation of `map` is about 7% slower than the zig version.